api/v1/model_handlers.go

```diff
@@ -272,6 +274,20 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Param(ws.QueryParameter("end", "End time for requested metric").DataType("string")).
 		Writes(MetricResult{}))
 
+	// The /sum/namespaces/{namespace-name}/pods/{pod-name}/metrics/{metric-name} endpoint exposes
+	// an aggregated metric for a list od pods of the model.
+	ws.Route(ws.GET("/sum/namespaces/{namespace-name}/pods/{pod-list}/metrics/{metric-name}").
```

Comment:
You're right, the `/pods/sum` subpath could cause issues in case there is pod named "sum" in that namespace.
 
A change in external result types is perfectly fine for this change.